Sources/SwiftDocCPluginUtilities/PluginFlags/SkipSynthesizedSymbolsFlag.swift

```diff
@@ -0,0 +1,26 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+extension PluginFlag {
+    /// Excludes synthesized symbols from the generated documentation.
+    ///
+    /// If `--skip-synthesized-symbols` is found, pages for synthesized symbols
+    /// will not be generated, reducing the archive file size by not emitting any `view-implementations`
+    /// folder.
+    static let skipSynthesizedSymbols = PluginFlag(
+        parsedValues: [
+            "--skip-synthesized-symbols"
+        ],
+        abstract: "Excludes synthesized symbols from the generated documentation",
+        description: """
+            Produces a DocC archive with a reduced file size when converted by omitting synthesized symbols.
```

Comment:
There are other reasons (besides reduced size on disk) that people might be interested in enabling this feature so I'm not sure it's worth mentioning that in the help text here.

```suggestion
            Experimental feature that produces a DocC archive without compiler synthesized symbols.
```

Reply:
Thanks, Ethan! 🙏 https://github.com/apple/swift-docc-plugin/pull/28/commits/b20147a2dab588c70a7b1d54a028d3132e01a86f